src/components/forms/DiskDeviceFormRoot.tsx

```diff
@@ -105,16 +107,23 @@ const DiskDeviceFormRoot: FC<Props> = ({
             readOnly: readOnly,
             overrideValue: formRootDevice?.pool,
             overrideForm: (
-              <StoragePoolSelector
-                project={project}
-                value={formRootDevice?.pool ?? ""}
-                setValue={(value) =>
-                  void formik.setFieldValue(`devices.${rootIndex}.pool`, value)
-                }
-                selectProps={{
-                  className: "u-no-margin--bottom",
-                }}
-              />
+              <>
+                <StoragePoolSelector
+                  project={project}
+                  value={formRootDevice?.pool ?? ""}
+                  setValue={(value) =>
+                    void formik.setFieldValue(
+                      `devices.${rootIndex}.pool`,
+                      value,
+                    )
+                  }
+                  selectProps={{
+                    className: isEditingInstance ? "" : "u-no-margin--bottom",
```

Comment:
We need the class also in the non-edit/create case

![image](https://github.com/user-attachments/assets/9e280e82-a08c-4f90-93a3-0dad17988ff9)

Reply:
aah yes, actually we need to hide the help text if we are not editing an instance. `u-no-margin--bottom` is only needed when there is no help text in order to reduce vertical spacing between the pool selector and the size input